imagersite/views.py

```diff
@@ -1,12 +1,23 @@
 from django.http import HttpResponse
 from django.template import loader
```

Comment:
Why not use `django.shortcuts.render` instead of doing the whole `loader.get_template --> template.render` sequence?